student_info/lehman_garrison.csv

```diff
@@ -0,0 +1,2 @@
+Name,Center,Research Focus,Fun Fact
+Lehman Garrison,CCA,Cosmology,"Hello, there!"
```

Comment:
"Hello there" could be more fun, but otherwise I think this is great. Is that how you spell "Cosmology"?

Reply:
Other people can also participate in the threaded conversation here!